src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,10 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user: { email, name } }) => (
+  <p>
+    {' Posted by  '}
```

Comment:
it's bad practice to adding spaces this way, better to use margins and padding for that